lemur/common/validators.py

```diff
@@ -138,3 +125,35 @@ def verify_private_key_match(key, cert, error_class=ValidationError):
     """
     if key.public_key().public_numbers() != cert.public_key().public_numbers():
         raise error_class("Private key does not match certificate.")
+
+
+def verify_cert_chain(certs, error_class=ValidationError):
+    """
+    Verifies that the certificates in the chain are correct.
```

Comment:
This is exactly the same function as used in #2262. It doesn't matter which one gets merged first.